Sources/Bot/Services/MergeService.swift

```diff
@@ -51,7 +51,7 @@ public final class MergeService {
         state.producer
             .combinePrevious()
             .startWithValues { old, new in
-                logger.log("♻️ [Merge Service] Did change state\n - 📜 \(old) \n - 📄 \(new)")
+                logger.log("♻️ Did change state\n - 📜 \(old) \n - 📄 \(new)")
```

Comment:
Why this cleanup? Feels useful in order to do a quick search when using `heroku logs` to find those messages though 😉

Reply:
This bot only manages the merge of pull requests now so this kind of filtering is no longer relevant as we don't have noisy neighbours anymore 😃

Reply:
Ah, good point!